src/fu-util-bios-attr.h

```diff
@@ -16,3 +16,5 @@ gboolean
 fu_util_bios_attr_matches_args(FwupdBiosAttr *attr, gchar **values);
 gboolean
 fu_util_get_bios_attr_as_json(gchar **values, GPtrArray *attrs, GError **error);
+GHashTable *
+fu_util_bios_attr_args_to_hash_table(gchar **input, GError **error);
```

Comment:
can we const this? then it's not looking like it's a `gchar *` that's being returned.

Reply:
sorry, i meant `fu_util_bios_attr_args_to_hash_table(const gchar **input, GError **error);` or `fu_util_bios_attr_args_to_hash_table(const gchar * const *input, GError **error);` if you want to be super correct.

Reply:
Nope. I tried but `g_strv_length` is very unhappy with this.